src/2dplane/GridStateSpace.hpp

```diff
@@ -19,6 +19,9 @@ class GridStateSpace : public PlaneStateSpace {
     bool stateValid(const Eigen::Vector2f &pt) const;
     bool transitionValid(const Eigen::Vector2f &from, const Eigen::Vector2f &to) const;
 
+    //adaptive stepsize control
+    Eigen::Vector2f intermediateState(const Eigen::Vector2f &source, const Eigen::Vector2f &target, float stepSize, float limit) const;
```

Comment:
Can you document here that adaptive stepsize control is used when the stepSize is < 0.